testing/it/ui-js/src/content/jcr_root/apps/core/wcm/tests/components/test-page-v2/customheaderlibs.html

```diff
@@ -14,4 +14,4 @@
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
 <sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"
-     data-sly-call="${clientlib.css @ categories=['core.wcm.components.search.v1', 'core.wcm.components.breadcrumb.v2']}"/>
+     data-sly-call="${clientlib.css @ categories=['core.wcm.components.search.v1', 'core.wcm.components.breadcrumb.v3']}"/>
```

Comment:
Is core.wcm.components.breadcrumb.v3 defined or we stick with core.wcm.components.breadcrumb.v2